source/includes/library-reference/_brain.html.md

```diff
@@ -13,73 +13,54 @@ such that ">>" indicates a decreasing order of precedence. Note that failure to
 BRAIN name in at least one of these places will result in a friendly error.
 
 ## Brain(config, name)
```

Comment:
Broadly, is it possible to have code snippet formatting in the section headers? It hurts my brain a tiny bit to see the heading number and the name of the constructor/method/field in the same font.